modules/dr-restoring-cluster-state.adoc

```diff
@@ -38,7 +38,7 @@ If you do not complete this step, you will not be able to access the control pla
 +
 This procedure assumes that you copied the `backup` directory containing the etcd snapshot and the resources for the static pods to the `/home/core/` directory of your recovery control plane host.
 
-. Stop the static pods on all other control plane nodes.
+. Stop the static pods on any other control plane nodes.
```

Comment:
@marun Is this step necessary if this procedure is only used when 2/3 control plane nodes are lost? 

Also, in the previous step to this one (step 2) in this procedure, should the step be updated to read "Establish SSH connectivity to the recovery control plane host." because presumably there are no other control plane hosts available other than the host used as the recovery host?

Reply:
I think 'any other control plane nodes' covers both the case of there being only the recovery control plane node and there being more control plane nodes.

I also don't think the previous step needs to be changed. If some control plane nodes are down, then there is no point in having ssh access to them.